docs/api.rst

```diff
@@ -4715,6 +4741,50 @@ Select
 
 .. autofunction:: nextcord.ui.select
```

Comment:
This should be modified to `string_select`, same with the other ones:
![image](https://user-images.githubusercontent.com/34910574/200666871-08161ff2-269d-4b1f-a9ce-19722d856339.png)